src/app/reskin/withdraw/components/stepper.tsx

```diff
@@ -0,0 +1,170 @@
+import { WithdrawalStatus } from "@/app/withdraw/[txid]/components/util";
+import { defineStepper, Stepper } from "@stepperize/react";
+import { useParams } from "next/navigation";
+import { useWithdrawalInfo } from "../hooks/use-withdrawal-info";
+import { useAtomValue } from "jotai";
+import { bridgeConfigAtom } from "@/util/atoms";
+import { HyperLink } from "../../components/stepper/deposit/status";
+import { CheckCircleIcon, XCircleIcon } from "@heroicons/react/24/outline";
+import { LoadingIndicator } from "../../assets/loading-indicator";
+import { getExplorerUrl } from "@/lib/get-explorer-url";
+import { getStacksNetwork } from "@/util/get-stacks-network";
+
+function SubmitWithdraw({}) {
+  return (
+    <span className="md:opacity-60 px-4 md:px-0">
+      How much BTC are you transferring back to BTC? Enter an amount that’s
+      above the dust requirement (546 sats)
+    </span>
+  );
+}
+
+function AddressDescription({}) {
+  return (
+    <span className="md:opacity-60 px-4 md:px-0">
+      BTC will be sent to a Bitcoin address. Connecting a wallet will auto-fill
+      this in, but feel free to submit another address.
+    </span>
+  );
+}
+
+export function TxPendingStatus({
+  stacksTxId,
+  bitcoinTxId,
+}: {
+  stacksTxId: string;
+  bitcoinTxId?: string;
+}) {
+  const { WALLET_NETWORK, PUBLIC_MEMPOOL_URL } = useAtomValue(bridgeConfigAtom);
+  return (
+    <div className="flex flex-col items-center justify-center mt-4 h-28">
+      <LoadingIndicator />
+      <div className="h-8 flex items-center text-xl leading-normal tracking-[-0.02em] text-center text-reskin-dark-gray dark:text-midGray uppercase mt-2 font-matter-mono">
+        Processing
+      </div>
+      <div>
+        ( Estimation{" "}
+        <HyperLink
+          href={
+            bitcoinTxId
+              ? `${PUBLIC_MEMPOOL_URL}/tx/${bitcoinTxId}`
+              : getExplorerUrl(stacksTxId, getStacksNetwork(WALLET_NETWORK))
+          }
+        >
+          5min
```

Comment:
nit: 5min is def wrong here since we need to wait 6 blocks, not sure the best way to handle this.

Reply:
I think we might need some feedback on this, same thing with deposits too